### PR TITLE
Port seo url improvements to v8

### PR DESCRIPTION
**Pull request by kaerdsar**
I had to merge all changes from v9 first to make the website_seo work. After that I was able to start checking each issue of v8. This pull request is only to test issue 36. Once this pull request passed the review I can do more pull requests for the other issues. (#36)

### CODE REVIEW

File: website_seo/i18n/de.po [hunk 1]
```diff
@@ -70,4 +70,21 @@ msgstr ""
...
+
+#. module: website_seo
+#: model:website.seo.information,information:website_seo.field_description_tip
+msgid "The meta description will be displayed on the Google search results pages. The meta description is a detailed characterization of the HTML page. It should encourage users to click on the search result (Call-to-Action). The length should be a maximum of 156 characters (including spaces)."
+msgstr "Die Meta Beschreibung wird auf den Google Suchergebnisseiten angezeigt. Die Meta Beschreibung ist eine nähere Charakterisierung der HTML Seite. Sie soll Benutzer animieren auf das Suchergebnis zu klicken (Call-to-Action). Die Länge sollte maximal 156 Zeichen (mit Leerzeichen) betragen."
+
+#. module: website_seo
+#. openerp-web
+#: code:addons/website_seo/static/src/xml/website_seo_robots.xml:8
```
Comment (rruebner) [inline website_seo/i18n/de.po]: @kaerdsar This translation doesn't work. The english part is shown. Tested in latest v8.

**Comment by rruebner** [comment]
Issues to solve:
- if I change the SEO url for a non default langauge in the promote panel the website menu entry isn't updated, e. g.:
  - I have two languages, english and german (default)
  - first I change the SEO url in german in the promote panel -> url and website menu entry are fine
  - second I change the SEO url in english in the promote panel -> url is fine but the website menu entry still has the german SEO url, it wasn't updated
- if I change the SEO url of blog.blog (except of "/blog", see #38) in the promote panel the website menu entry isn't updated, it is still /blog/1
- (minor issue) if I change the SEO url of blog.post I am redirected to "/blog_post_seo_url" instead of "/blog_blog_seo_url/blog_post_seo_url"

**Comment by kaerdsar** [comment]
I found in v8 the Promote Panel can be opened from a non default language page, even the menu Content to create pages. In this case we don't need the language selector in Promote Panel for v8. The configuration can be done by change the page language.

**Comment by rruebner** [comment]
Review was fine :+1:

**Pull request merged**